### PR TITLE
5_9_3 5_9_2b

### PR DESCRIPTION
**Pull request by Avuxo**
all that マイナス点 stuff feels a lot more obvious now. Maybe I'm becoming nasubrained

### CODE REVIEW

File: script/Arcueid/Day 5/05_00_ARC05_9_2B.txt [hunk 1]
```diff
@@ -26,22 +27,27 @@ The earth itself erupted. You'd be forgiven for thinking a bomb had gone off.
...
-The power of an overwhelming number of fragments. If Vlov's cold air is like a blizzard, then what Arcueid did is like an earth tsunami.
+The power of an overwhelming number of fragments.
+ If Vlov's cold air is like a blizzard,
+ then what Arcueid did is like an earth tsunami.
 }
 [sha:afef4e00d09328d323a22a176d37427de0f746d6]{
 -- Page 661, Offset 10427.
 -- 飛翔する氷の槍など、この壁の如き土砂の前では飛沫にすぎない。
-A flying ice spear is nothing more than a flung droplet of water before this wall-like agglomeration of earth and sand.
+A flying ice spear is like flicking droplet of water before this wall of earth and sand.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_9_2B.txt:37]: flicking `a` droplet, but you'd also need to alter the 'before this wall' a bit I think. If we're gonna tone this one down then maybe just
> A flying ice spear is nothing more than a droplet of water before this...

File: script/Arcueid/Day 5/05_00_ARC05_9_2B.txt [hunk 2]
```diff
@@ -66,23 +73,25 @@ In the face of the towering wave, the man managed to regain his lapsed sense of
...
+// Reified verb from above
 No, he regained it two minutes ago to be exact.
 }
 [sha:e47fc39eb406a0747647ec50cab6374fbf065c09]{
 -- Page 664, Offset 10435.
 -- 常に寒波に<侵|おか>され正気を無くした筈の脳が、あの一瞬―――目前に迫る人間から飛び退いた時から、正常に稼働しようと努めていた。
-His mind, which should have lost all sanity from the constant barrage of cold, has been trying to function normally ever since the moment he jumped away from the human closing in before him.
+His mind, which should have lost all sanity from the constant barrage of cold,
+ has been trying to function normally ever since that moment―――
+ The moment he jumped away from the human closing in on him.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_9_2B.txt:84]: Continuation of line after sentence ended with em-dash is uncommon in the script, intentional? We usually have them either be terminal or no spaces either side (in which case probably no caps on `the moment`

File: script/Arcueid/Day 5/05_00_ARC05_9_2B.txt [hunk 3]
```diff
@@ -92,27 +101,33 @@ Vestiges of the time when he was normal.
 [sha:e7307f7f73537b5f3349684e6089d128f3bf44fc]{
 -- Page 665, Offset 10439.
 -- 中世の暗黒時代に培った騎士としての闘争経験が、全身に巡る呪いを打ち払った。
-His battle experience as a knight, forged in the Dark Ages of antiquity, has routed the curse that bound him hand and foot.
+His battle experience as a knight,
+ forged in the Dark Ages,
+ has pushed aside the curse that bound him hand and foot.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_9_2B.txt:106]: kinda like `routed`.

File: script/Arcueid/Day 5/05_00_ARC05_9_2B.txt [hunk 4]
```diff
@@ -92,27 +101,33 @@ Vestiges of the time when he was normal.
...
 -- たった二十年程度の矜持が、千年を超える狂気に勝ったのだ。
-The pride and dignity of a mere twenty years, triumphant over the madness that transcends thousands.
+The pride and dignity of a mere twenty years,
+ triumphant over the madness that transcends thousands.
 }
 [sha:3e0a372963bb9d733231bbb905fd8570a6b75282]{
 -- Page 666, Offset 10441.
 -- それがどのような奇跡なのか、彼はよく判っている。
-He knows well the kind of miracle this is.
+He knows well how what kind of miracle this is.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_9_2B.txt:117]: drop how?

File: script/Arcueid/Day 5/05_00_ARC05_9_2B.txt [hunk 5]
```diff
@@ -92,27 +101,33 @@ Vestiges of the time when he was normal.
...
+Moments like this come and go.
+ Nothing more than a trick of the imagination before killing that human.
+ Simply a dream of an afterlife that will never come.
 }
 [sha:fbf4165628e00cb99fed2458d278ef2ea93906f4]{
 -- Page 666, Offset 10443.
 -- 戦闘に興じるなぞ―――そんな人間らしい<歓|よろこ>びは、あと数秒で凍り付く。
-Never indulge yourself during combat―――Such human-like joy will freeze over in a matter of seconds.
+Never indulge yourself during combat―――
+Such human-like joy will freeze over in a matter of seconds.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_9_2B.txt:130]: Wouldn't capitalize after conjoined em-dash

File: script/Arcueid/Day 5/05_00_ARC05_9_2B.txt [hunk 6]
```diff
@@ -481,12 +504,15 @@ I've completely given up on responding to him at this point.
 [sha:d34f43aea8ce075329bb89da3577d632105bc2db]{
 -- Page 689, Offset 10516.
 -- 俺の思考速度では１秒に一つの判断しかできない。
-I can only make one decision per second with my current speed of thought.
+// overclocked shiki
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_9_2B.txt:507]: get some ln2 on those meodp

File: script/Arcueid/Day 5/05_00_ARC05_9_2B.txt [hunk 7]
```diff
@@ -661,7 +706,7 @@ This is fine.
 [sha:122e7ae036f6925857c6f7ff85721ec3ff76f36a]{
 -- Page 700, Offset 10552.
 -- ここで脳を串刺しにされても、きっとまったく気がつかない。
-I'm sure that if he skewered my brain here, I wouldn't notice it at all.
+I'm sure that if he skewered my brain here, I'm sure I wouldn't even notice.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_9_2B.txt:709]: Drop second I'm sure

File: script/Arcueid/Day 5/05_00_ARC05_9_2B.txt [hunk 8]
```diff
@@ -727,12 +776,17 @@ Having abandoned all thought, I understand through instinct.
...
+If lines represent the death of life,
+ then that point represents the death of existence.
 }
 [sha:df8c498b97cf72836e228c3f1dfc166cd82e0a56]{
 -- Page 704, Offset 10566.
 -- どのような不死身、どのような怪物であれ一刺しで“殺す”、この世でもっともおぞましい、宇宙の終わりに他ならない。
-Any immortal being, any monster, could be 'killed' by a single strike. It is none other than the end of the universe, the most repulsive thing in the world.
+Any immortal being,
+ any monster,
+ could be 'killed' with a single strike.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_9_2B.txt:787]: can be killed?

File: script/Arcueid/Day 5/05_00_ARC05_9_3.txt [hunk 9]
```diff
@@ -67,7 +70,7 @@ There's no time to spare.
 [sha:a27c2665c1de07f7f32961b9366b73790f60ddfa]{
 -- Page 804, Offset 10845.
 -- 『ヴローヴを蔦で縛ってほしい』というジェスチャーをしながら叫ぶ。
-I shout while gesturing my intention to tie Vlov up with the ivy.
+I shout and gesturing my intention to tie Vlov up with the ivy.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_9_3.txt:73]: > and _gesture_ that I want her to tie Vlov down with the ivy.

**Review by rschlaikjer — APPROVED** [review]
otsukare

File: script/Arcueid/Day 5/05_00_ARC05_9_2B.txt (hunk 2 above)
Comment (Avuxo) [reply script/Arcueid/Day 5/05_00_ARC05_9_2B.txt:84]: unintentional. artifact of a different structure i was toying with

Comment (Avuxo) [reply script/Arcueid/Day 5/05_00_ARC05_9_2B.txt:84]: wait, no. reading back over, this dash series was specifying the moment. It was supposed to be 2x dashes and no caps.

File: script/Arcueid/Day 5/05_00_ARC05_9_2B.txt (hunk 3 above)
Comment (Avuxo) [reply script/Arcueid/Day 5/05_00_ARC05_9_2B.txt:106]: can't say I agree. "routed" without an explicit direction sounds kinda weird to me

Comment (rschlaikjer) [reply script/Arcueid/Day 5/05_00_ARC05_9_2B.txt:106]: I feel like you can rout the enemy without direction, but don't care that much.

**Pull request merged**